src/sql/base/browser/ui/taskbar/actionbar.ts

```diff
@@ -128,15 +143,87 @@ export class ActionBar extends ActionRunner implements IActionRunner {
 		this._actionsList = document.createElement('ul');
 		this._actionsList.className = 'actions-container';
 		this._actionsList.setAttribute('role', 'toolbar');
+		this._actionsList.id = 'actions-container';
 		if (this._options.ariaLabel) {
 			this._actionsList.setAttribute('aria-label', this._options.ariaLabel);
 		}
 
 		this._domNode.appendChild(this._actionsList);
 
+		if (this._collapseOverflow) {
+			this._overflow = document.createElement('ul');
```

Comment:
Why are you creating a list element here? Doesn't a regular div suffice? And why wouldn't this just go on the parent container to begin with instead of creating and inserting a whole new container?

Reply:
And to begin with - is there a reason you're not using the ContextMenuService?

https://github.com/microsoft/azuredatastudio/blob/master/src/vs/platform/contextview/browser/contextMenuService.ts#L42